internal/commands/scan.go

```diff
@@ -638,6 +643,10 @@ func scanCreateSubCommand(
 	createScanCmd.PersistentFlags().Bool(commonParams.ReportSbomFormatLocalFlowFlag, false, "")
 	_ = createScanCmd.PersistentFlags().MarkHidden(commonParams.ReportSbomFormatLocalFlowFlag)
 
+	createScanCmd.PersistentFlags().String(commonParams.SCSRepoTokenFlag, "", "GitHub token to be used with SCS engines")
```

Comment:
confirm those texts with Fatima / Eli / your PM, this text will be displayed to the customer

Reply:
I believe this was agreed between everyone, but @arturfalcao did this part. I am double checking